roles/jd-server/Cargo.toml

```diff
@@ -16,8 +16,8 @@ binary_sv2 = { version = "^1.0.0", path = "../../protocols/v2/binary-sv2/binary-
 buffer_sv2 = { version = "^1.0.0", path = "../../utils/buffer" }
 codec_sv2 = { version = "1.0.0", path = "../../protocols/v2/codec-sv2", features = ["noise_sv2"] }
 const_sv2 = { version = "^1.0.0", path = "../../protocols/v2/const-sv2" }
-network_helpers = { version = "1.0.0", path = "../../utils/network-helpers", features = ["with_tokio"] }
-noise_sv2 = { version = "1.0.0", path = "../../protocols/v2/noise-sv2" }
+network_helpers_sv2 = { version = "1.0.0", path = "../roles-utils/network-helpers", features = ["with_tokio"] }
+noise_sv2 = { version = "1.1.0", path = "../../protocols/v2/noise-sv2" }
```

Comment:
Why you bumped this?

Reply:
`noise_sv2` is on `v1.1.0`

https://github.com/stratum-mining/stratum/blob/23227812bad2973e3169c3e664853bba782d021a/protocols/v2/noise-sv2/Cargo.toml#L3

I'm not sure why `jd-server` even compiled before, but should be fixed.